cobbler/template_api.py

```diff
@@ -79,15 +71,15 @@ def __init__(self, **kwargs):
             "#end if",
             "#end def",
         ]) + "\n")
-        super(Template, self).__init__(**kwargs)
+        super(CobblerTemplate, self).__init__(**kwargs)
```

Comment:
```suggestion
        super().__init__(**kwargs)
```
Should do the same thing according to the [docs](https://docs.python.org/3/library/functions.html#super) and is Python 3 style.